migrations/20181021111949-create-ticket.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Tickets', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      price: {
```

Comment:
We should keep consistency between files. If in this file we use 
```
price: {
  type: Sequelize.DOUBLE
},
```

In another file (`server/models/airline.js` for example) we use this format
```
price: Sequelize.DOUBLE
```

If we need to asign the type, I suggest we do it shorthard everywhere and this way we keep it clean and consistent